### ISSUE TITLE
Emmet is installed but it doesn't work.  Brackets on Linux Mint 19.2 Cinnamon.

### ISSUE THREAD

**Issue by ui-coder** [adobe/brackets#14958]
Hi everyone. I'm a fairly new user of Linux OS and I faced a problem trying to fix it.
Here's my issue! The Emmet extension is installed on Brackets but it doesn't work at all after I tried to change the _`keymap.json`_ and fix the "Wrap with Abbreviation" shortcut. After that (I'm sure though) Emmet on Brackets (Linux Mint 19.2 Cinnamon) toolbar did not even open by clicking on it to see the menu. Any suggestions could be a help. Thanks to y'all.

**Comment by ui-coder** [comment]
Hi everyone. I'm a fairly new user of Linux OS and I faced a problem trying to fix it.
Here's my issue! The Emmet extension is installed on Brackets but it doesn't work at all after I tried to change the keymap.json and fix the "Wrap with Abbreviation" shortcut. After that (I'm sure though) Emmet on Brackets (Linux Mint 19.2 Cinnamon) toolbar did not even open by clicking on it to see the menu. Any suggestions could be a help. Thanks to y'all.
